src/Microsoft.Health.Dicom.Api/Features/Exceptions/ExceptionHandlingMiddleware.cs

```diff
@@ -106,6 +106,7 @@ private IActionResult MapExceptionToResult(Exception exception)
             case InstanceAlreadyExistsException:
             case ExtendedQueryTagsAlreadyExistsException:
             case ExtendedQueryTagsOutOfDateException:
+            case ExistingUpdateOperationException:
```

Comment:
Instead of adding more things here, just have `ExistingUpdateOperationException` and `ExistingReindexOperationException` derive from `InvalidOperationException` which is already tracked here (and is semantically appropriate).

Reply:
InvalidOperationException sends a 400 whereas the other two should be a conflict.

Reply:
I am unsure if a conflict makes the most sense. A conflict implies that a resource at the URL already exists, but for update I don't see any semantic reason why you couldn't submit two updates in quick succession other than performance (and perhaps ordering complexities on the backend). If the second update request will succeed if I just wait long enough, a "conflict" doesn't seem like the reason we rejected it.

Reply:
As per our current design, we allow only one request to be queued and there is no check on the study instance ids that we receive in the request. I feel a conflict on the same update request makes more sense at this time, than a bad request.

Reply:
I suppose we can argue that the Conflict is appropriate because the state of the resource is in flux during update, and an additional update may conflict with what is being applied. In this case, can you please update re-indexing's exception to perhaps share some common exception that we can catch in the `ExceptionHandlingMiddleware`. Or better yet create a generic shared exception that is used in both scenarios and get rid of the operation-specific exception. E.g. `OngoingOperationException` or something like that.

Reply:
Created a new parent `ExisitngOperationException` class and added to exception middleware.